components/network/openvpn/Makefile

```diff
@@ -37,7 +39,6 @@ CONFIGURE_OPTIONS += CPPFLAGS="$(CPPFLAGS)"
 CONFIGURE_OPTIONS += GREP=/usr/gnu/bin/grep
 CONFIGURE_OPTIONS += --disable-static
 CONFIGURE_OPTIONS += --enable-shared
-CONFIGURE_OPTIONS += --enable-password-save
```

Comment:
It seems to be enabled by default - https://github.com/OpenVPN/openvpn/commit/1e9c1f09cba95ebf72083c746cf847056a61c761